File: server/SvyU.Models/QuestionType.cs
using System;
using System.Collections.Generic;
using System.Text;
using Newtonsoft.Json;
using Newtonsoft.Json.Converters;
using Newtonsoft.Json.Serialization;

namespace SvyU.Models
{
    public enum QuestionType
    {
        Single = 0,
        Multiple = 1,
        Text = 2
    }
}

File: android-client/app/src/main/java/mg/studio/android/survey/models/QuestionType.java
package mg.studio.android.survey.models;

import androidx.annotation.NonNull;

/**
 * Represents the type of a question.
 */
public enum QuestionType {
    /**
     * Represents a question that accepts single choice-based response.
     */
    Single("single"),
    /**
     * Represents a question that accepts multiple choice-based response.
     */
    Multiple("multiple"),
    /**
     * Represents a question that accepts arbitrary text input response.
     */
    Text("text");

    private final String value;
    QuestionType(final String value) {
        this.value = value;
    }

    @NonNull
    @Override
    public String toString() {
        return value;
    }
}
